src/tpg_timer.c

```diff
@@ -750,15 +752,23 @@ static cmdline_parse_token_string_t cmd_show_timer_statistics_t_statistics =
     TOKEN_STRING_INITIALIZER(struct cmd_show_timer_statistics_result, statistics, "statistics");
 static cmdline_parse_token_string_t cmd_show_timer_statistics_t_details =
     TOKEN_STRING_INITIALIZER(struct cmd_show_timer_statistics_result, details, "details");
+static cmdline_parse_token_string_t cmd_show_timer_statistics_T_port_kw =
```

Comment:
Minor: For uniformity I would rename these tokens to:
`cmd_show_timer_statistics_t_port_kw`
`cmd_show_timer_statistics_t_port`